### ISSUE TITLE
[OUDS] Missing colors of elements in utilities/position documentation

### ISSUE THREAD

**Issue by vprothais** [Orange-OpenSource/Orange-Boosted-Bootstrap#3359]
### Prerequisites

- [x] I have [searched](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [x] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [x] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

In the utilities/position document, elements that are positionned doesn't have an existing color, so they are not visible.

### Reduced test cases

Can be seen in documentation

### What operating system(s) are you seeing the problem on?

macOS

### What browser(s) are you seeing the problem on?

Firefox

### What version of Boosted are you using?

1.0.0
